### PR TITLE
Adding left and right chevrons to carousel

### PR DESCRIPTION
**Pull request by lukeamarsh**
https://appearhere.atlassian.net/browse/LFR-1709

- Added left and right arrow buttons on carousel
- Fixed security vulnerabilities with `serialize-javascript` and `mem` packages
- Fixed glitch issues with hover states on cards when used in the carousel
- Removed SectionHeader from carousels and replaced with h2

### CODE REVIEW

File: packages/core/src/components/MobileCarousel/MobileCarousel.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
 import PropTypes from 'prop-types';
 import React, { Children } from 'react';
-import shortid from 'short-id';
-import SectionHeader from '../Type/SectionHeader/SectionHeader';
+import nanoid from 'nanoid';
 
 import css from './MobileCarousel.css';
 
 const MobileCarousel = ({ children = [], title }) => {
+  var id = nanoid();
```
Comment (Denyer) [inline packages/core/src/components/MobileCarousel/MobileCarousel.js]: the id needs to be generated in the loop for each element

File: packages/core/src/components/Carousel/Carousel.js [hunk 2]
```diff
@@ -1,48 +1,77 @@
...
+    if (!props.useCenterControls || (!props.infinite && slideIndex === 0)) return null;
+    return (
+      <BtnContainer onClick={() => setSlideIndex(slideIndex - 1)} className={cx(css.control, css.prev)}>
+        <Icon className={css.prevIcon} name="chevron" />
+        <ScreenReadable>{props.accessibilityPrevLabel}</ScreenReadable>
+      </BtnContainer>
+    )
+  }
+
+  const renderCenterRightcontrols = () => {
```
Comment (Denyer) [inline packages/core/src/components/Carousel/Carousel.js]: should be camelCase (`renderCenterRightControls`)

File: packages/core/src/components/Carousel/Carousel.js [hunk 3]
```diff
@@ -1,48 +1,77 @@
...
+              <Icon className={cx(css.chevron, css.nextIcon)} name="chevron" />
+              <ScreenReadable>{props.accessibilityNextLabel}</ScreenReadable>
+            </BtnContainer>
           </div>
         )}
       </div>
     );
   };
 
+  const renderCenterLeftcontrols = () => {
```
Comment (Denyer) [inline packages/core/src/components/Carousel/Carousel.js]: should be camelCase (`renderCenterLeftControls`)

File: packages/core/src/components/Carousel/Carousel.js [hunk 4]
```diff
@@ -1,48 +1,77 @@
...
 
 import css from './Carousel.css';
 
 const Carousel = props => {
+  const id = nanoid();
   const [slideIndex, setSlideIndex] = useState(0);
 
-  const renderDesktopControls = () => {
+  const renderTopRightControls = () => {
+    if (!props.useTopRightControls) return null;
```
Comment (Denyer) [inline packages/core/src/components/Carousel/Carousel.js]: I think it reads better to put this condition outside of the function and only call the function when necessary:
```jsx
{props.useTopRightControls && renderTopRightControls()}
```

File: packages/core/src/components/Carousel/Carousel.js [hunk 5]
```diff
@@ -1,48 +1,77 @@
...
+              <ScreenReadable>{props.accessibilityNextLabel}</ScreenReadable>
+            </BtnContainer>
           </div>
         )}
       </div>
     );
   };
 
+  const renderCenterLeftcontrols = () => {
+    if (!props.useCenterControls || (!props.infinite && slideIndex === 0)) return null;
```
Comment (Denyer) [inline packages/core/src/components/Carousel/Carousel.js]: Again I think it reads better to move the condition `!props.useCenterControls` outside of the function and only call when necessary:
```jsx
{props.useCenterControls && renderCenterLeftControls()}
```

File: packages/core/src/components/MobileCarousel/MobileCarousel.js (hunk 1 above)
Comment (lukeamarsh) [reply packages/core/src/components/MobileCarousel/MobileCarousel.js]: 🤦‍♂ totally missed this. good spot!

File: packages/core/src/components/Carousel/Carousel.js (hunk 2 above)
Comment (lukeamarsh) [reply packages/core/src/components/Carousel/Carousel.js]: 👍

File: packages/core/src/components/Carousel/Carousel.js (hunk 3 above)
Comment (lukeamarsh) [reply packages/core/src/components/Carousel/Carousel.js]: 👍

File: packages/core/src/components/Carousel/Carousel.js (hunk 4 above)
Comment (lukeamarsh) [reply packages/core/src/components/Carousel/Carousel.js]: yep agreed. Reads much better

File: packages/core/src/components/Carousel/Carousel.js [hunk 6]
```diff
@@ -1,51 +1,79 @@
...
       <NukaCarousel slideIndex={slideIndex} {...props}>
-        {props.children.map((slide) => (
-          <div key={shortid.generate()} className={css.slideInner}>
-            {slide}
-          </div>
-        ))}
+        {props.children.map((slide) => {
+          const id = nanoid();
+          return (
+            <div key={id} className={css.slideInner}>
```
Comment (Denyer) [inline packages/core/src/components/Carousel/Carousel.js]: the declaration of id is redundant:
```suggestion
            <div key={nanoid()} className={css.slideInner}>
```

**Pull request merged**